test/unit/HDWallet.spec.ts

```diff
@@ -7,6 +8,10 @@ const publicKeyStr =
     "0x95c259ecefa8e60edb099a6568887dd9ba35afa75062d01a12fcc5a40d402eca57afcdc1cff7e34fbcd84dfea0848bb9";
 
 describe(".generate()", () => {
+    beforeAll(async () => {
+        await initBLS();
+    });
```

Comment:
Add this to setupFile:
https://jestjs.io/docs/en/configuration.html#setupfiles-array

Reply:
It doesn't work with async code, there are some hacks that don't look nice